models/user.php

```diff
@@ -21,4 +30,33 @@ public function register(){
 		}
 		return;
 	}
+	public function login(){
+		// Sanitize POST
+		$post = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+
+		$password = md5($post['password']);
+
+		if($post['submit']){
```

Comment:
Same as previous comment, do not nest in if here, just do the check and return:
```suggestion
		if(!$post['submit']) {
		    return;
		}
```